services/web/server/src/simcore_service_webserver/director_v2/_handlers.py

```diff
@@ -92,13 +96,28 @@ async def start_computation(request: web.Request) -> web.Response:
             )
         )
 
+    # Get wallet information
+    wallet_info = None
+    project_wallet = await projects_api.get_project_wallet(
+        request.app, project_id=project_id
+    )
+    if project_wallet and os.environ.get("WEBSERVER_DEV_FEATURES_ENABLED", False):
```

Comment:
I am not sure I get the thing here.
so if here is a wallet but you are in DEV_FEATURE mode there is no wallet? is this correct?